datastore/common/objects/OperatingSystem.cpp

```diff
@@ -84,6 +96,26 @@ namespace netmeld::datastore::objects {
         ;
   }
 
+  std::string
+  OperatingSystem::ConvertVectorToPostgresArray(const std::vector<std::string>& vec) {
```

Comment:
Do we want to store the hotfixes in a concatenated string? If so that will work but I thought initially we could store them in an array, to have more flexibility over retrieval?

Reply:
https://www.postgresql.org/docs/current/arrays.html#ARRAYS-INPUT

Reply:
No we don't want a concatenated string.  Also, not sure why you are referencing PostgreSQL documentation in this case as that's not the layer you are working at.  It is libpqxx documentation you should be referencing (as I mentioned in my prior comment).

Regardless, I am marking this as resolved since it is outdated and will comment on the new code.